src/test/regress/expected/alter_database_propagation.out

```diff
@@ -1,38 +1,30 @@
 set citus.log_remote_commands = true;
 set citus.grep_remote_commands = '%ALTER DATABASE%';
--- since ALLOW_CONNECTIONS alter option should be executed in a different database
--- and since we don't have a multiple database support for now,
--- this statement will get error
-alter database regression ALLOW_CONNECTIONS false;
-ERROR:  ALLOW_CONNECTIONS is not supported
 alter database regression with CONNECTION LIMIT 100;
-NOTICE:  issuing ALTER DATABASE regression WITH  CONNECTION LIMIT 100;
+NOTICE:  issuing ALTER DATABASE regression WITH  CONNECTION_LIMIT 100;
```

Comment:
This seems an unintended change. It works both ways, but the docs say `CONNECTION LIMIT` (with a space instead of underscore)